src/recipes/fadeTransition.ts

```diff
@@ -1,5 +1,18 @@
 import { defineSlotRecipe } from '@pandacss/dev'
 
+export type FadeTransitionRecipeVariant = {
+  duration:
+    | 'fast'
+    | 'slow'
+    | 'medium'
+    | 'distractionFreeTyping'
+    | 'nodeFadeIn'
+    | 'nodeFadeOut'
+    | 'nodeDissolve'
+    | 'disappearingUpperRight'
+    | 'disappearingLowerLeft'
+}
```

Comment:
This type declaration is confusing, because `FadeTransitionRecipeVariant` (also `SlideTransitionRecipeVariant`) is already imported and used prior to this PR:

https://github.com/cybersemics/em/blob/aa6016e018a758bf219ca152f7fbca9643611107/src/components/FadeTransition.tsx#L3

Is there a reason that this type needs to be explicitly defined and exported here? It seems that our Panda configuration already magically generates it. This is also the only explicitly exported type in `src/recipes`, so it doesn't seem to be following the project's usual patterns.

Reply:
This was due to a temporary build error, and I thought explicitly exporting the type might help. But you're right, it was unnecessary. I've removed it now.